src/app/view/components/login-as/login-as.spec.ts

```diff
@@ -115,15 +115,12 @@ describe('LoginAsComponent', () => {
     });
 
     it('should call redirect to the first porlet when login as happen', () => {
```

Comment:
I think we don't need this here now... we need to test this in the navigation service.

What we need to test here is that the login as method is called.

And the same thing goes for the logout as.